covfee/client/admin/hit_block/journey_buttons.tsx

```diff
@@ -30,6 +30,32 @@ export const JourneyRow = ({
 }: JourneyRowProps) => {
   const { addChats } = React.useContext(chatContext)
   const { getUrl } = useJourneyFns(journey)
+  const [annotator_data, setAnnotatorData] = React.useState<Annotator>(null)
+  const [show_annotator_data, setShowAnnotatorData] =
```

Comment:
same, please use camelCase standard

Reply:
Fixed in 06bbf53eba006fb394fef90cc275f8062d7a3762